wagtail_transfer/static_src/components/ContentImportForm/index.js

```diff
@@ -124,9 +151,13 @@ export default function ImportContentForm({
 
         <li className="transfer numbered">
           <div class="transfer list-container">
-            <h2>Select destination parent page</h2>
+            <h2>
+              {!alreadyExistsAtDestination
+                ? 'Select destination parent page'
+                : 'This page already exists at the destination, and will be updated.'}
+            </h2>
           </div>
-          {sourcePage ? (
+          {sourcePage && alreadyExistsAtDestination == null ? (
```

Comment:
I think this should be `alreadyExistsAtDestination === false`? as we want to hide this if it is `null` or `true`